apps/selvbetjening-ui/src/inntektsjustering/InnledningTilInntektsjustering.tsx

```diff
@@ -1,64 +1,68 @@
-import { Alert, Bleed, BodyShort, Button, GuidePanel, Heading, Hide, HStack, Link, VStack } from '@navikt/ds-react'
+import { Alert, Bleed, Button, GuidePanel, Heading, Hide, HStack, Select, VStack } from '@navikt/ds-react'
 import { ArrowRightIcon } from '@navikt/aksel-icons'
 import { SkjemaProgresjon } from './components/SkjemaProgresjon.tsx'
 import { VarigLoonnstilskuddIcon } from './icons/VarigLoonnstilskuddIcon.tsx'
-import { useNavigate } from 'react-router-dom'
+import { Navigate, useNavigate } from 'react-router-dom'
+import useSWR, { SWRResponse } from 'swr'
+import { apiURL } from '../utils/api.ts'
+import { SanityRikTekst } from '../common/sanity/SanityRikTekst.tsx'
+import { useState } from 'react'
+import { Spraak } from '../types/spraak.ts'
 
 export const InnledningTilInntektsjustering = () => {
     const navigate = useNavigate()
 
-    return (
-        <HStack justify="center" padding="8">
-            <VStack gap="6" maxWidth="42.5rem">
-                <HStack gap="4" align="center">
-                    <Hide below="md">
-                        <VarigLoonnstilskuddIcon />
-                    </Hide>
-                    <Heading size="xlarge" level="1">
-                        Meld fra om inntekt til neste år
-                    </Heading>
-                </HStack>
+    const [valgtSpraak, setValgtSpraak] = useState<Spraak>(Spraak.BOKMAAL)
 
-                <SkjemaProgresjon aktivtSteg={1} />
+    const { data, error }: SWRResponse<never[], boolean, boolean> = useSWR(
```

Comment:
`never[]` er nok uheldig på sikt, men det er nok ikke noe man trenger  å fikse opp i med en gang. i stedet for never har vi vel egentlig noe ala 

```
type SanityData = Record<SanityTekster, Spraak>[]

type SanityTekster = 'tittel' | 'info' | 'hovedinnhold'

type Spraak = {
  'EN': string,
  'NB': string,
  'NN': string
}

const data: SanityData = [{
  'tittel': {
    'EN': 'Title',
    'NB': 'Tittel',
    'NN': 'Tittel',
  },

  'info': {
    'EN': 'Info',
    'NB': 'Info',
    'NN': 'Info',
  },

  'hovedinnhold': {
    'EN': 'Main content',
    'NB': 'Hovedinnhold',
    'NN': 'Hovedinnhold',
  }
}]
```

og jeg vet ikke hvor lett dette er å få generert fra sanity-schema'et vårt :p

Men noe bedre enn `never` må vi nok ha en gang